docs/supported.md

```diff
@@ -515,14 +515,14 @@ For color definition, KaTeX color functions will accept the standard HTML [pred
 ||||
 |:------------------------------|:------------------------------|:-----
 |$\mathrm{Ab0}$ `\mathrm{Ab0}`  |$\mathbf{Ab0}$ `\mathbf{Ab0}`  |$\mathit{Ab}$ `\mathit{Ab}`
-|$\textrm{Ab0}$ `\textrm{Ab0}`  |$\textbf{Ab0}$ `\textbf{Ab0}`  |$\textit{Ab}$ `\textit{Ab}`
-|$\rm Ab0$ `\rm Ab0`            |$\bf Ab0$ `\bf Ab0`            |$\it Ab$ `\it Ab`
-|$\textnormal{Ab0}$ `\textnormal{Ab0}`|$\bold{Ab0}$ `\bold{Ab0}`|$\Bbb{AB}$ `\Bbb{AB}`
-|$\text{Ab0}$ `\text{Ab0}`      |$\boldsymbol{Ab}$ `\boldsymbol{Ab}`|$\mathbb{AB}$ `\mathbb{AB}`
-|$\mathsf{Ab0}$ `\mathsf{Ab0}`  |$\bm{Ab}$ `\bm{Ab}`            |$\frak{Ab0}$ `\frak{Ab0}`
-|$\textsf{Ab0}$ `\textsf{Ab0}`  |$\mathtt{Ab0}$ `\mathtt{Ab0}`  |$\mathfrak{Ab0}$ `\mathfrak{Ab0}`
-|$\sf Ab0$ `\sf Ab0`            |$\texttt{Ab0}$ `\texttt{Ab0}`  |$\mathcal{AB0}$ `\mathcal{AB0}`
-|                               |$\tt Ab0$ `\tt Ab0`            |$\mathscr{AB}$ `\mathscr{AB}`
+|$\mathnormal{Ab0}$ `\mathnormal{Ab0}`|$\textbf{Ab0}$ `\textbf{Ab0}`  |$\textit{Ab}$ `\textit{Ab}`
+|$\textrm{Ab0}$ `\textrm{Ab0}`  |$\bf Ab0$ `\bf Ab0`            |$\it Ab$ `\it Ab`
+|$\rm Ab0$ `\rm Ab0`            |$\bold{Ab0}$ `\bold{Ab0}`|$\Bbb{AB}$ `\Bbb{AB}`
+|$\textnormal{Ab0}$ `\textnormal{Ab0}`|$\boldsymbol{Ab}$ `\boldsymbol{Ab}`|$\mathbb{AB}$ `\mathbb{AB}`
+|$\text{Ab0}$ `\text{Ab0}`      |$\bm{Ab}$ `\bm{Ab}`            |$\frak{Ab0}$ `\frak{Ab0}`
+|$\mathsf{Ab0}$ `\mathsf{Ab0}`  |$\mathtt{Ab0}$ `\mathtt{Ab0}`  |$\mathfrak{Ab0}$ `\mathfrak{Ab0}`
+|$\textsf{Ab0}$ `\textsf{Ab0}`  |$\texttt{Ab0}$ `\texttt{Ab0}`  |$\mathcal{AB0}$ `\mathcal{AB0}`
+|$\sf Ab0$ `\sf Ab0`            |$\tt Ab0$ `\tt Ab0`            |$\mathscr{AB}$ `\mathscr{AB}`
```

Comment:
Any guesses why I'm not seeing these changes on https://deploy-preview-1700--katex.netlify.com/docs/supported.html ? Is there something wrong with our Netlify builds?

Reply:
@edemaine It's because versioned docs are not updated. You can check them in https://deploy-preview-1700--katex.netlify.com/docs/next/supported.html.

Reply:
Of course! Thanks.

Now I see a bug, though: the `0` in `\mathnormal` his rendering in the incorrect font.

![image](https://user-images.githubusercontent.com/2218736/46355968-d5f27780-c62f-11e8-8fd4-0a4852f14635.png)

According to inspect, it's rendering with the HTML `<span class="mord mathcal">0</span>` whereas the letters are rendering as e.g. `<span class="mord mathdefault">A</span>`.

Reply:
Oops, I guess that's exactly what LaTeX does.  Sorry, `\mathnormal` does not do what I thought it did!

```latex
\mathnormal{Ab0}\mathit{Ab0}
```
![image](https://user-images.githubusercontent.com/2218736/46356554-fb33b580-c630-11e8-9eda-a4e65cd6e5df.png)

I do get the feeling that there's too much space after the `A` in all other KaTeX examples though...

Reply:
Comparing more with LaTeX, everything seems like a good match. Which your texcmp outputs confirm. 😄 

I do remain confused about what the initial/default mode of LaTeX is. I thought it was `\mathnormal`, but that differs on numerals. Is there a name for it?

Reply:
`\mathdefault`?  I don't even though if that's a thing or not, but there must be some way of getting back to that mode from inside of `\mathnormal`.